client/src/components/Question/index.jsx

```diff
@@ -4,9 +4,10 @@ import QuestionList from '../QuestionList'
 
 const Question = (props) => (
   <div>
-  {props.data.slice(0,1).map((question) => {
+  {props.data.slice(0,1).map((question, index) => {
```

Comment:
since these are questions, i bet they have question Ids, which would be unique enough and better than using an index.  future PR please

Reply:
update key prop to use question id, thanks Phil!